### PR TITLE
fix: Since OOB policies have the same policy id, creating a new key t…

### PR DESCRIPTION
**Pull request by rramalinga**
…o uniquely to identify policy id

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: specs/cloudpolicy.spec [hunk 1]
```diff
@@ -46,6 +46,14 @@ attributes:
...
     stored: true
 
+  - name: key
+    description: |-
+      Unique key which is a combination of tenant id and policy id to identify the
+      policy.
+    type: string
+    exposed: true
+    stored: true
+
```
Comment (dstiliadis) [inline specs/cloudpolicy.spec:56]: key should be indexed.

**Review by dstiliadis — DISMISSED** [review]
Key must be an index to avoid any performance issues on upserts

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-oob-policies",
    "component": "backend",
    "pr-id": "1431",
    "commit-sha": "a8f05b87da4a0a3a5e9112d96b35a99fe48891b2",
    "pipeline": "master"
  },
  {
    "project": "fix-oob-policies",
    "component": "gaia",
    "pr-id": "731",
    "commit-sha": "f05540a0d42cdbc1d1385e65ec681e22d6ec17db",
    "pipeline": "master"
  }
]
```

**Review by t00f — APPROVED** [review]

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-oob-policies",
    "component": "backend",
    "pr-id": "1431",
    "commit-sha": "6b864f8408483a1ff48309c87ef5e53cede89747",
    "pipeline": "master"
  },
  {
    "project": "fix-oob-policies",
    "component": "gaia",
    "pr-id": "731",
    "commit-sha": "f05540a0d42cdbc1d1385e65ec681e22d6ec17db",
    "pipeline": "master"
  }
]
```

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-oob-policies",
    "component": "backend",
    "pr-id": "1431",
    "commit-sha": "d30ba198c80329cbdd9ed5f0a8b34f4783266a13",
    "pipeline": "master"
  },
  {
    "project": "fix-oob-policies",
    "component": "gaia",
    "pr-id": "731",
    "commit-sha": "f05540a0d42cdbc1d1385e65ec681e22d6ec17db",
    "pipeline": "master"
  }
]
```

**Comment by kevincantu** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1431",
    "commit-sha": "d30ba198c80329cbdd9ed5f0a8b34f4783266a13",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "731",
    "commit-sha": "f05540a0d42cdbc1d1385e65ec681e22d6ec17db",
    "pipeline": "master"
  }
]
```

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-oob-policies",
    "component": "backend",
    "pr-id": "1431",
    "commit-sha": "d30ba198c80329cbdd9ed5f0a8b34f4783266a13",
    "pipeline": "master"
  },
  {
    "project": "fix-oob-policies",
    "component": "gaia",
    "pr-id": "731",
    "commit-sha": "f05540a0d42cdbc1d1385e65ec681e22d6ec17db",
    "pipeline": "master"
  }
]
```

**Pull request merged**